src/plotting/plot.component.tsx

```diff
@@ -34,6 +40,12 @@ const Plot = (props: PlotProps) => {
     YAxesSettings,
     XAxis,
     canvasRef,
+    gridVisible,
+    axesLabelsVisible,
+    xMinimum,
+    xMaximum,
```

Comment:
I thought we had min/max settings in `XAxisSettings`/`YAxesSettings`? Makes fewer props to pass around, although I don't object to having them explicitly separate, but if the only option in the `(X/Y)AxisSettings` is `scale` then we should extract that too

Reply:
You're right and I'd totally forgotten. I'm going to try things out by using the min and max fields in `AxisSettings`, it may well make things better

Reply:
Tried it out but it proved too difficult to cohesively manage the state updates and callbacks. So I've extracted scale out to be its own type in https://github.com/ral-facilities/operationsgateway/pull/92/commits/3f43316c621f9d64db0329117b90118ee17a0f8d